tutor/src/screens/assignments/details.js

```diff
@@ -56,9 +68,9 @@ const Details = observer(({ ux }) => {
             <HintText>(Apply a pre-set submission and grading policy template)</HintText>
           </RowLabel>
           <div>
-            <Select name="grading_template_id">
+            <StyledSelect name="grading_template_id">
               {ux.gradingTemplates.map(t => <option key={t.id} value={t.id}>{t.name}</option>)}
-            </Select>
+            </StyledSelect>
             <PreviewTooltip template={ux.gradingTemplate} />
```

Comment:
I think we should ask UX what to do when there's only a single grading template for that type of assignment.  In that case it seems like we should just show it's name as text rather than a select that doesn't do anything

Reply:
That's a great thing to point out, I'll bring that up.